packages/mds-provider-service/@types/index.ts

```diff
@@ -0,0 +1,38 @@
+import type { DomainModelCreate } from '@mds-core/mds-repository'
+import type { RpcEmptyRequestContext, RpcServiceDefinition } from '@mds-core/mds-rpc-common'
+import { RpcRoute } from '@mds-core/mds-rpc-common'
+import type { Nullable, UUID } from '@mds-core/mds-types'
+
+export const PROVIDER_TYPES = ['mds_micromobility', 'gbfs_micromobility'] as const
```

Comment:
can you expand this to:
const PROVIDER_TYPES = <const>['mds_micromobility', 'mds_taxi', 'mds_tnc', 'gbfs_micromobility']
? 
I'm preparing for the medium-far-off "taxi&tnc" mode of conductor